src/main/java/com/revature/registry/model/Project.java

```diff
@@ -19,37 +19,38 @@
 import org.hibernate.annotations.LazyCollection;
 import org.hibernate.annotations.LazyCollectionOption;
 
+/**
+ * A {@link Project} is one of the Capstone Projects managed by the Center of Excellence. A {@link Project} is composed
+ * of many {@link Iteration}s, can only be in one {@link Status} at a time, and progresses through each {@link Phase}s
+ * in the order presented. A {@link Project} is searchable and filterable by {@link Tag}s.
+ */
+
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 @Entity
 public class Project {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private int id;
 
     private String name;
+
     @ManyToOne
     @JoinColumn(name = "status")
     private Status status;
-    @ManyToOne
-    @JoinColumn(name = "stack")
-    private Stack stack;
+
     private String description;
 
     @ManyToOne
     @JoinColumn(name = "created_by")
-    private Account account;
+    private Account createdBy;
```

Comment:
Will this create issues as members of CoE join and leave? Can we instead have an associated "owner" that can be transferred? Or is this not an issue at all?

Reply:
You're right, I think that would cause issues. Will modify.